### DISCUSSION TITLE
el wtanaia tv algeria

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#283] [Q&A]
plz extract the m3u8 link of this channel from algeria
https://elwataniatv.dz/%d9%85%d8%a8%d8%a7%d8%b4%d8%b1/

**Comment by Sphinxroot** [comment]
topUrl
https://elwataniatv.dz/%d9%85%d8%a8%d8%a7%d8%b4%d8%b1/

referrer	
https://player.castr.com/live_3949b8b03cc511edacc24fd73f3ea4db

Stream Detected
https://stream.castr.com/62c18c3f14d09a0b7e5355a5/live_3949b8b03cc511edacc24fd73f3ea4db/index.m3u8?check=1
https://stream.castr.com/62c18c3f14d09a0b7e5355a5/live_3949b8b03cc511edacc24fd73f3ea4db/tracks-v1a1/mono.m3u8
https://backend.dna-delivery.com/router/v2/10001f6b-63ee-4e8d-9aa7-71b9c44a40a3/content/stream.castr.com%2F62c18c3f14d09a0b7e5355a5%2Flive_3949b8b03cc511edacc24fd73f3ea4db%2Findex.m3u8
https://stream.castr.com/62c18c3f14d09a0b7e5355a5/live_3949b8b03cc511edacc24fd73f3ea4db/index.m3u8

#EXTINF:-1 ,el wtanaia tv
#EXTVLCOPT:http-referrer=https://player.castr.com/live_3949b8b03cc511edacc24fd73f3ea4db
https://stream.castr.com/62c18c3f14d09a0b7e5355a5/live_3949b8b03cc511edacc24fd73f3ea4db/index.m3u8

#EXTINF:-1 ,el wtanaia tv (Kodi) (480p)
https://stream.castr.com/62c18c3f14d09a0b7e5355a5/live_3949b8b03cc511edacc24fd73f3ea4db/index.m3u8|Referer=https://player.castr.com/live_3949b8b03cc511edacc24fd73f3ea4db


i suspect link change in this part /62c18c3f14d09a0b7e5355a5/   if not  it can be added

**Comment by Alstruit** [comment]
See issue #9105
